hack/observability/prometheus/values.yaml

```diff
@@ -14,6 +14,8 @@ pushgateway:
 # Scrape metrics from deployed providers
 extraScrapeConfigs: |
   - job_name: 'capi-providers'
+    # 15s is a bit often for production but helps to get metrics quicker for development.
```

Comment:
rebase errata?

Reply:
The idea was to make the metrics that we pick up with CR v0.14.5 better accessible in the dev env.

But I can move it to a separate PR if we prefer

Reply:
got it, IMO including this change in this PR makes sense

Reply:
Okay. I'm really fine either way